PICMI_Python/diagnostics.py

```diff
@@ -329,3 +329,48 @@ def __init__(self, grid, num_snapshots, dt_snapshots, data_list=None,
         self.name = name
 
         self.handle_init(kw)
+
+class PICMI_ParticleBoundaryScrapingDiagnostic(_ClassWithInit) :
+    """
+    Defines the particle diagnostics that are absorbed at the boundaries, throughout the simulation.
```

Comment:
Here's a suggested edit of this documentation
`Defines the particle diagnostics that are used to collect the particles that are absorbed at the boundaries, throughout the simulation.`